benchmarks/DeepLearning/Layers/SelfAttention/buddy_selfattention_import.py

```diff
@@ -0,0 +1,60 @@
+import os
+import torch
+from torch._inductor.decomposition import decompositions as inductor_decomp
+
+from buddy.compiler.frontend import DynamoCompiler
+from buddy.compiler.graph import GraphDriver
+from buddy.compiler.graph.transform import simply_fuse
+from buddy.compiler.ops import tosa
+
+class SelfAttentionLayer(torch.nn.Module):
+    def __init__(self, embed_dim, num_heads):
+        super(SelfAttentionLayer, self).__init__()
+        self.attention = torch.nn.MultiheadAttention(embed_dim, num_heads, batch_first=True)
+    
```

Comment:
MultiheadAttention or SelfAttention? I think they are not the same?

Reply:
It is selfattention because it is used in this selfattention way
`attn_output, _ = self.attention(x, x, x)`